### ISSUE TITLE
Stalebot should not close feature requests

### ISSUE THREAD

**Issue by pataquets** [syl20bnr/spacemacs#15790]
I know it can be helpful to tame the amount of issues.
But closing FRs is counterproductive. It can lead to filing duplicate FRs, and losing previous discussions.
I'd suggest to make stalebot leave alone issues labelled as FRs and optionally add "help wanted" labels for FRs not having maintainers on it, which is understandable. Also, such a label sends a clear message about expectations.

As an example, here's an issue worthwhile IMO which I was about to file again: #9451.
EDIT: Almost happened again: #9925.

**Comment by smile13241324** [comment]
Good point, I think this bot is controlled by a file in the repo, care to make a PR?

**Comment by pataquets** [comment]
@smile13241324 Done on #15806.

**Comment by pataquets** [comment]
BTW, I would like to ask for #9451 to be labelled as such and kept open.

**Comment by pataquets** [comment]
As #9451 is reopened, I've noticed that it still has the "stale" label. AFAICT, it should not, should it?
Anything else needed in the PR? I'm afraid stalebot might target the above mentioned issue again.

**Comment by smile13241324** [comment]
Please check my comment in the PR it seems I have not enough rights to change the workflow.

**Comment by pataquets** [comment]
Should I file a new instance of issue #9925 or could it be reopened?

If staleboting is done due to excess FRs, they could be tagged as 'PR welcome' to indicate that no maintainer is commited/interested in coding it. This would help in filtering out issues labeled "FR+PR welcome", but would leave them open to anybody to pick up. Losing FRs and their discussion sounds as valuable info and effort lost.

@TheBB @lebensterben @JAremko @syl20bnr : your opinion is appreciated. There is a PR at #15806 as suggested by @smile13241324 that went closed due to lack of feedback. I think some policing is needed in how to handle FRs, which does not mean that they should take for granted that will be done, but at least, not lost. Some smart usage of labels would help, I guess, but I can't tell for sure due to lack of info.

**Comment by pataquets** [comment]
Happened again for same issue... https://github.com/syl20bnr/spacemacs/issues/9925#issuecomment-2105956177
The stalebot closed issue is well relevant, but it might be completely overlooked by others interested.

@TheBB @lebensterben @JAremko @syl20bnr: Sorry to spam you again, but feedback is much needed from sb who has the decision power into this matter. As stated in the comment above, I even went thru sending a PR, which ended up closed.
Who is to ask who has the decision power into this matter? What are the drawbacks, if any?

**Comment by smile13241324** [comment]
I don't think this can be changed right now as far as I see I have not the rights to change it

**Comment by McPhale** [comment]
Closing an issue due to permissions doesn't seem (to me) like a valid reason to close it

**Comment by smile13241324** [comment]
Point taken, issue is reopened
